### DISCUSSION TITLE
Network Replication for Unreal

### DISCUSSION THREAD

**Discussion by chriswingler** [JSBSim-Team/jsbsim#665] [Q&A] [answered]
Hi, does JSBSIM for Unreal support network replication?  Thanks.

**Comment by gallonmate** [comment, marked as answer]
Nope but it could be added like you would for any Unreal Project. Theoretically it should work as FlightGear uses JSBSim in networked multiplayer. 
The Unreal plugin and Reference project currently uses JSBSim similar to how FlightGear works:

The Game Engine starts by Initializing JBSSim

Then the game engine loops during play:
- Sends inputs to JBSSim
- Tell JBSim to run a simulation frame(or multiple frames)
- Request output from JBSSim
- Repeat

**Comment by BariyCO3** [comment]
> Hi, does JSBSIM for Unreal support network replication? Thanks.

Hi, have you succeded in solving this problem?
